EventListener/SecurityListener.php

```diff
@@ -65,7 +66,12 @@ public function onKernelController(FilterControllerEvent $event)
         }
 
         if (!$this->language->evaluate($configuration->getExpression(), $this->getVariables($request))) {
-            throw new AccessDeniedException(sprintf('Expression "%s" denied access.', $configuration->getExpression()));
+            if ($statusCode = $configuration->getStatusCode()) {
+                throw new HttpException($statusCode, $configuration->getMessage());
```

Comment:
We need a default message in case someone only modified the status code.

Reply:
I would just use `Access denied` as a default message.